test/acvp_test.c

```diff
@@ -1292,6 +1292,84 @@ static int rsa_decryption_primitive_test(int id)
 }
 #endif /* OPENSSL_NO_RSA */
 
+static int drbg_test(int id)
+{
+    OSSL_PARAM params[3];
+    EVP_RAND *rand = NULL;
+    EVP_RAND_CTX *ctx = NULL, *parent = NULL;
+    unsigned char returned_bits[64];
+    const size_t returned_bits_len = sizeof(returned_bits);
+    unsigned int strength = 256;
+    const struct drbg_st *tst  = &drbg_data[id];
+    int res = 0;
+
+    /* Create the seed source */
+    if (!TEST_ptr(rand = EVP_RAND_fetch(libctx, "TEST-RAND", "-fips"))
```

Comment:
This test only runs against the fips module.. so "-fips" is optional.

Reply:
It's required.  The test RNG isn't FIPS and won't be found without this.